### PR TITLE
Add capability via NextcloudClient

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Signed-off-by: tobiasKaminsky <tobias@kaminsky.me>

### CODE REVIEW

File: src/androidTest/java/com/owncloud/android/GetCapabilitiesTest.java [hunk 1]
```diff
@@ -82,4 +81,45 @@ public void testGetRemoteCapabilitiesOperationEtag() {
...
+        assertTrue(result.isSuccess());
+        assertTrue(result.getData() != null && result.getData().size() == 1);
+
+        OCCapability capability = (OCCapability) result.getData().get(0);
+
+        Assert.assertSame(capability.getRichDocuments(), CapabilityBooleanType.FALSE);
+
+        Assert.assertFalse(capability.getDirectEditingEtag().isEmpty());
+
+        // TODO assert basic capabilities
```
Comment (AndyScherzinger) [inline src/androidTest/java/com/owncloud/android/GetCapabilitiesTest.java:101]: Just a reminder for future work to be done or an open issue that you want to implement before merging this PR?

**Comment by AndyScherzinger** [comment]
Just a small question

File: src/androidTest/java/com/owncloud/android/GetCapabilitiesTest.java (hunk 1 above)
Comment (tobiasKaminsky) [reply src/androidTest/java/com/owncloud/android/GetCapabilitiesTest.java]: Better do it now :)

**Comment by tobiasKaminsky** [comment]
@dependabot-bot rebase

**Pull request merged**